.gitlab/e2e.yml

```diff
@@ -314,6 +314,26 @@ new-e2e-orchestrator-main:
 
   allow_failure: true #TODO: Remove when https://github.com/DataDog/datadog-agent/pull/22113 is merged
 
+new-e2e-apm-dev:
+  extends: .new_e2e_template
+  rules:
+    - !reference [.if_run_e2e_tests]
+    - !reference [.on_dev_branch_manual]
+  needs:
+    - qa_branch_agent
+    - qa_branch_dca
+    - qa_branch_dogstatsd
```

Comment:
These are copy/pasted from the existing jobs - can the reviewers please confirm it's correct?

Reply:
Required if you have tests based on the agent docker image (docker, kindvm, ecs or eks). If you're using the agent on host, this is not needed.

Reply:
Thanks, I'm actually using the agent on docker in these tests.